core/trino-main/src/test/java/io/trino/sql/planner/optimizations/TestEliminateSorts.java

```diff
@@ -56,20 +55,20 @@ public class TestEliminateSorts
             "lineitem",
             ImmutableMap.of(QUANTITY_ALIAS, "quantity"));
 
-    @Test
-    public void testEliminateSorts()
-    {
-        @Language("SQL") String sql = "SELECT quantity, row_number() OVER (ORDER BY quantity) FROM lineitem ORDER BY quantity";
-
-        PlanMatchPattern pattern =
-                output(
-                        window(windowMatcherBuilder -> windowMatcherBuilder
-                                        .specification(windowSpec)
-                                        .addFunction(functionCall("row_number", Optional.empty(), ImmutableList.of())),
-                                anyTree(LINEITEM_TABLESCAN_Q)));
-
-        assertUnitPlan(sql, pattern);
-    }
+//    @Test
+//    public void testEliminateSorts()
+//    {
+//        @Language("SQL") String sql = "SELECT quantity, row_number() OVER (ORDER BY quantity) FROM lineitem ORDER BY quantity";
+//
+//        PlanMatchPattern pattern =
+//                output(
+//                        window(windowMatcherBuilder -> windowMatcherBuilder
+//                                        .specification(windowSpec)
+//                                        .addFunction(functionCall("row_number", Optional.empty(), ImmutableList.of())),
+//                                anyTree(LINEITEM_TABLESCAN_Q)));
+//
+//        assertUnitPlan(sql, pattern);
+//    }
```

Comment:
Remove this commented-out code. If and when we reintroduce that behavior, we can add a test as appropriate.

Reply:
Oops, I have forgotten to remove these lines after commented out. 🙇‍♂️